Cesium3DTilesSelection/src/ImplicitOctreeLoader.cpp

```diff
@@ -147,7 +147,8 @@ CesiumAsync::Future<TileLoadResult> requestTileContent(
           CesiumGltfReader::GltfReaderOptions gltfOptions;
           gltfOptions.ktx2TranscodeTargets = ktx2TranscodeTargets;
           gltfOptions.applyTextureTransform = applyTextureTransform;
-          GltfConverterResult result = converter(responseData, gltfOptions);
+          GltfConverterResult result =
+              converter(responseData, gltfOptions, nullptr); // XXX
```

Comment:
Did you leave the XXX here as a reminder for something?

Reply:
Yes, and will add comment. I wasn't sure how to calculate the tile transform for the implicit loaders.